unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/AssetManager/Texture/AssetPromise_Texture.cs

```diff
@@ -84,6 +88,7 @@ protected override void OnLoad(Action OnSuccess, Action<Exception> OnFail)
                 byte[] decodedTexture = Convert.FromBase64String(url.Substring(PLAIN_BASE64_PROTOCOL.Length));
                 asset.texture = new Texture2D(1, 1);
                 asset.texture.LoadImage(decodedTexture);
+                asset.texture = TextureHelpers.ClampSize(asset.texture, DataStore.i.textureSize.generalMaxSize.Get());
```

Comment:
shouldn't that be
```suggestion
                asset.texture = TextureHelpers.ClampSize(asset.texture, maxTextureSize);
```
?

Reply:
🤦

Reply:
> please, make sure this PR isn't breaking scenes using images as an atlas :)
@pbosio

I set a hugue capping value (2048) just because of that use case.